platform/accessibility/guidance/post-launch-audit-processes.md

```diff
@@ -0,0 +1,19 @@
+# Accessibility Post-Launch Audit Processes
+Post-launch audit processes are designed to evaluate applications and static content using a wider set of devices than the [accessibility staging review](./staging-review-processes.md). The accessibility specialist(s) will also schedule your application or content for a VA 508 office quick test.
+
+You can [request a full audit](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=1Copenut%2C+jenstrickland&labels=508%2FAccessibility%2C+product+support&template=full-accessibility-and-508-office-audit.md&title=Full+Accessibility+%26+508+Office+Audit+%5BFeature-Name%5D) when your application is relatively code-stable in production. Ideally all issues identified during the staging review have been fixed, but your accessibility specialist will communicate known issues to the VA 508 office.
+
+## How The Audit is Different from a Staging Review
+The post-launch audit will focus on manual testing with mobile and assistive technology:
+
+* Windows 7/10 IE11 + JAWS
+* Windows 7/10 Chrome + JAWS
+* Windows 7/10 NVDA + Firefox
+* MacOS Safari + VoiceOver
+* iOS and Android mobile devices as available
+* Browsers such as AVG or Waterfox. Additional browsers will be tested as time allows, and may change depending on analytics data.
+
+## Issue Gathering and Triage
+* Issues will be collected as issues in a new ZenHub epic. The accessibility specialist will assign this epic to the person who requested the staging review.
+* Issues will be triaged according to their severity. Teams should review the [accessibility defect severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md) if they have questions about issue severity or impact.
```

Comment:
It may help teams if we clarify how issues are to be cleared. Once there is a fix in, they should leave a comment on the ticket to let the a11y specialist know it is ready for validation. If there are issues still, the a.s. provides a comment with details/guidance. If the issue is now remedied, the a.s. leaves a comment that they validated the issue is clear. Do we need to @ a team member, or do they receive notifications on the ticket they're assigned / contributed to?

Reply:
I offered some general guidance about closing tickets. I don't want to get too caught up on an exact methodology; some teams it'll work better for them to close, others the specialists will. I'm inclined to maintain that flexibility.